ops/envs/batch-submitter.env

```diff
@@ -4,6 +4,7 @@ ETH_NETWORK_NAME=clique
 LOG_LEVEL=debug
 BATCH_SUBMITTER_LOG_LEVEL=debug
 BATCH_SUBMITTER_LOG_TERMINAL=true
+BATCH_SUBMITTER_MIN_L1_TX_SIZE=32
```

Comment:
Is this in bytes?

Reply:
yes it is

Reply:
and it also includes the function selector

Reply:
Maybe include the unit with the config option, like metrics?
`BATCH_SUBMITTER_MIN_L1_TX_BYTES`
`BATCH_SUBMITTER_POLL_INTERVAL_MS`